recipes/minmea/all/conandata.yml

```
@@ -0,0 +1,9 @@
+sources:
+  "cci.20230620":
```

```suggestion
  "0.3.0.cci.20230620":
```
Should apply the new naming scheme for `cci.` versions.